apps/demos/src/app/app.module.ts

```diff
@@ -22,7 +22,7 @@ import { HomeComponent } from './features/home/home.component';
       useValue: environment,
     },
     {
-      provide: RX_ANGULAR_CONFIG,
+      provide: RX_ANGULAR_RENDER_STRATEGIES_CONFIG,
```

Comment:
we could prefix such consts with just `RXA_` which would decrease the length of the name quite a bit: `RXA_RENDER_STRATEGIES_CONFIG`.

We could also think of not postfixing it with CONFIG => `RXA_RENDER_STRATEGIES`

what do u say?